src/launcher/StracciatellaLauncher.cc

```diff
@@ -118,7 +118,12 @@ StracciatellaLauncher::StracciatellaLauncher() {
       o->end();
       Fl_Group::current()->resizable(o);
     } // Fl_Tabs* o
-    stracciatellaLauncher->size_range(465, 300);
+    { ja2JsonPathOutput = new Fl_Output(0, 300, 465, 25);
+      ja2JsonPathOutput->tooltip("ja2.json contains the stracciatella engine options");
+      ja2JsonPathOutput->color(FL_BACKGROUND_COLOR);
+      ja2JsonPathOutput->align(Fl_Align(FL_ALIGN_LEFT|FL_ALIGN_INSIDE));
+    } // Fl_Output* ja2JsonPathOutput
+    stracciatellaLauncher->size_range(465, 325);
```

Comment:
What good does displaying the path to ja2.json do? The launcher exists in great part to avoid the need to know about this file, plus it has only two keys, both of which are configurable through the launcher.

Reply:
For power users, `ja2.json` is essentially a file representation of the launcher options. (or the launcher a GUI for the file)

It just makes sense to me, since there is no help menu in the launcher.

Reply:
Must be the launcher then, since we don't save command line options. But using it that way you lose the help — the default has a comment inside, the launcher has tooltips, so I don't understand the argument.

Reply:
btw, there was some talk about extra search paths here: #290

Reply:
I never saw the default comment because I used the launcher to launch the game (that's when the file is saved) way before learning of the file.

I don't see an appropriate place where this path could go as a tooltip and I would lose the ability to copy the path. And if this was a tooltip, shouldn't the game settings path also be a tooltip?

Reply:
My argument is that people using the launcher don't need to know about the file at all. #692 is a reminder there's some work to be done for the users that skip the launcher (but can wait if we're swapping the VFS layer).

Also, what would happen if the path was longer than the textbox size?

@selaux @JAsmine-ja2 we need more opinions. :abcd:

Reply:
When the path is longer you can scroll left and right, like a normal input/output field, or just select everything by double clicking or ctrl+a and copy. (that's an output field, like the game settings)

Reply:
Uhuu, now I noticed... when I edit that file in a text editor it usually shows the path of the file one way or another, in my current one it's "ja2.json (/home/hp/.ja2)".

I view the launcher as a specialized editor for that file, so that explains why I think it fits.
Now it makes me want to add a save button, a reload button, and show '*' when the settings are modified but not saved, but I'll wait for your feedback first. xD

Reply:
Implemented it since selaux didn't comment on this.